README.md

```diff
@@ -1,9 +1,30 @@
 # VSCode XML Client
 
+## Description
+
+This VSCode extension provides support during the creation of XML documents
+
+## Features
```

Comment:
those are LSP features. Users don't care about those. You need to translate to plain English